quickstarts/azure/azure-functions-workflow/config.yml

```diff
@@ -1,3 +1,4 @@
+id: 099258cc-d981-40d4-907c-3b2881595735
```

Comment:
Was this addition intentional?

Reply:
Nope. No idea where that came from or why my commit message is incorrect... I'll fix it.